spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/mail/MailProperties.java

```diff
@@ -61,6 +62,11 @@
 	 */
 	private Map<String, String> properties = new HashMap<String, String>();
 
+	/**
+	 * JNDI location of the session.
+	 */
+	private String jndiName;
+
```

Comment:
Mail Session JNDI name. When set and JNDI is present, takes precedence to manual configuration.